python/pyarrow/compute.py

```diff
@@ -595,6 +595,32 @@ def bottom_k_unstable(values, k, sort_keys=None, *, memory_pool=None):
     return call_function("select_k_unstable", [values], options, memory_pool)
 
 
+def random(n, *, initializer='system', options=None, memory_pool=None):
```

Comment:
The automatic wrapper generation doesn't work for this kernel anymore because the `n` here is passed as the batch length of an empty `ExecBatch` now. Maybe it could be fixed to work but I don't know